.vib/kibana/cypress/cypress/integration/utils.js

```diff
@@ -7,14 +7,14 @@ export const skipTheWelcomeScreen = () => {
     closeThePopups();
     cy.get("body").then(($body) => {
         if ($body.text().includes('Explore on my own')) {
-            cy.get('button[data-test-subj="skipWelcomeScreen"]').should('be.visible').click();
+            cy.get('button[data-test-subj="skipWelcomeScreen"]').click();
         }
     })
 }
 
 export const closeThePopups = () => {
     cy.get('[data-test-subj="kbnLoadingMessage"]').should('not.exist');
```

Comment:
This is needed since we are waiting for the loading page to disappear before moving forward.

Reply:
👍 As the default behavior of `get` is to ensure that the element `exists`, we should not need to add a comment here.

Reply:
Please observe that the assertion is `not.exists`
I am waiting for the message **to disappear**, in order to interact with **other elements.**